server/apps/immich/src/api-v1/album/album-repository.ts

```diff
@@ -39,7 +39,7 @@ export class AlbumRepository implements IAlbumRepository {
   ) {}
 
   async create(ownerId: string, createAlbumDto: CreateAlbumDto): Promise<AlbumEntity> {
-    return await getConnection().transaction(async (transactionalEntityManager) => {
```

Comment:
It doesn't, I just removed it because in these case is not useful. When using await what it does is, it resolves the promise, so we now have the "album entity", then it wraps it in another resolved promise which is then returned.

Without the await it just returns the promise. A good practice is to have the return values defined in the function (don't let TypeScript infer), in this case `Promise<AlbumEntity>`, so whatever you do inside you must still return that type or TypeScript will complain.